question/bank/previewquestion/templates/preview_question.mustache

```diff
@@ -53,16 +63,13 @@
         {{/redirect}}
     </div>
 </form>
-{{{techinfo}}}
-{{{exporttoxml}}}
```

Comment:
why is this removed? its a feature

Reply:
It wasn't part of the wireframe, I put it back.